app/Http/Controllers/MembersController.php

```diff
@@ -467,4 +455,16 @@ public function transfer($id, Request $request)
 
         return view('members.transfer', compact('enquiry', 'invoice_number', 'invoiceCounter', 'member_code', 'memberCounter', 'member_number_mode', 'invoice_number_mode'));
     }
+
+    /**
+     * @param \Illuminate\Http\Request $request
+     * @return string
+     */
+    private function drpPlaceholder(Request $request)
+    {
+        if ($request->has('drp_start') and $request->has('drp_end')) {
```

Comment:
Why did you change the operator here from ||  to && ?

Reply:
it is still exact same thing.

if we are negate a condition we should also convert and to or and vice versa.
```php
if( $a or $b) {
 echo 'hi';
}else {
 echo 'bye';
}

if( !$a and !$b) {

 echo 'bye';
}else {
 echo 'hi';
}
```

Reply:
I know that is the same thing, maybe I should be more clear with the question: "what's the benefit between one way or another?"

Reply:
the conversion is and strait forward  thing.
and it is done automatically by IDE features .

Reply:
I just thought it reads better in this way.
Any way feel free to reject or edit PRs.
I just wanted to see it the maintainers are happy with contributions or not.

Reply:
Fair enough. I don't have to reject or edit the PRs just want to know the reasons and you provide them. 👍

Reply:
@ludo237 thanks for checking in the changes. 

@imanghafoori1 thanks for drying up the repetitive code. 

`And` seems to be perfect here as we want both the values.